### PR TITLE
[google][ios] Make provider="google" use same region format as MapKit

### PR DESCRIPTION
**Pull request by gilbox**
This effects events on iOS: the `onRegionChange` event as well as `onRegionChangeComplete` (the latter is currently only available with google).

gb-region

@spikebrehm @lelandrichardson

### CODE REVIEW

File: ios/AirGoogleMaps/AIRGoogleMap.m [hunk 1]
```diff
@@ -12,15 +12,38 @@
 #import "RCTConvert+MapKit.h"
 #import "UIView+React.h"
 
-id cameraPositionAsJSON(GMSCameraPosition *position) {
-  // todo: convert zoom to delta lat/lng
+id regionAsJSON(MKCoordinateRegion region) {
   return @{
-           @"latitude": [NSNumber numberWithDouble:position.target.latitude],
-           @"longitude": [NSNumber numberWithDouble:position.target.longitude],
-           @"zoom": [NSNumber numberWithDouble:position.zoom],
```
Comment (spikebrehm) [inline ios/AirGoogleMaps/AIRGoogleMap.m]: Is this a breaking change, because `zoom` is no longer passed through?

**Comment by spikebrehm** [comment]
Nice, seems legit 👍

File: ios/AirGoogleMaps/AIRGoogleMap.m (hunk 1 above)
Comment (spikebrehm) [inline ios/AirGoogleMaps/AIRGoogleMap.m]: Although, this was released in an rc.0, so we can make a "breaking" change still.

Comment (gilbox) [inline ios/AirGoogleMaps/AIRGoogleMap.m]: yep ( ◠‿◠ )

**Pull request merged**